web/src/locales/fr.json

```diff
@@ -29,6 +29,10 @@
     "areas": "Zones",
     "about": "À propos"
   },
+  "app-banner": {
+    "cta": "Télécharger l'app",
+    "description": "Données d'électricité en direct"
+  },
```

Comment:
Just bumped the internal Slack thread on this, I still think this is too long:

![Screenshot 2024-09-16 at 15 03 17](https://github.com/user-attachments/assets/987aa8f2-4afc-41e3-bbcb-3a463ae1b70a)

Reply:
Still not getting any responses on the thread. According to ChatGPT:
> In the French App Store, the download button usually says "Obtenir", which means "Get." 
> In the French version of the Google Play Store, the download button typically says "Installer", which means "Install."

I was able to confirm the [Play Store](https://play.google.com/store/search?q=electricity%20maps&c=apps&hl=fr), and I'm trying to confirm the App Store too

Reply:
![Uploading Screenshot 2024-09-17 at 12.08.02.png…]()

Reply:
[French support page with screenshots](https://www.google.com/imgres?q=ouvrir%20app%20store&imgurl=https%3A%2F%2Fcdsassets.apple.com%2Flive%2F7WUAS350%2Fimages%2Fitunes%2Flocale%2Ffr-fr%2Fios-16-iphone-13-pro-app-store-account-purchased.png&imgrefurl=https%3A%2F%2Fsupport.apple.com%2Ffr-fr%2F108091&docid=ZvUacex4Vz3GwM&tbnid=oxm2gYXrbpn_6M&vet=12ahUKEwia4JjH38mIAxUQSPEDHa-4FzwQM3oECH4QAA..i&w=700&h=680&hcb=2&ved=2ahUKEwia4JjH38mIAxUQSPEDHa-4FzwQM3oECH4QAA)

Reply:
[German support page with screenshots](https://support.apple.com/de-de/108091)

Reply:
Hey @phiphou, could you help out on this one? (and thanks for your other translation PRs recently 🙏)

Reply:
Sure, but would you prefer the App store or the Play store one ?

I would opt for "Installer l'application".

Tell me and I'll make the related PR.